app.json

```diff
@@ -4,7 +4,8 @@
   "website": "https://www.codetriage.com/",
   "repository": "https://github.com/codetriage/codetriage",
   "scripts":{
-    "postdeploy":"rake db:schema:load db:migrate db:seed"
+    "postdeploy":"DISABLE_DATABASE_ENVIRONMENT_CHECK=1
+ rake db:schema:load db:migrate db:seed"
```

Comment:
This shouldn't be required. Are you getting errors?

I wrote that feature. When the database is empty the environment check should basically not care what you do to it.